test/binaryen.js/custom-section.js.txt

```diff
@@ -10,7 +10,6 @@ int main() {
   std::map<size_t, BinaryenEventRef> events;
   std::map<size_t, BinaryenExportRef> exports;
   std::map<size_t, RelooperBlockRef> relooperBlocks;
-  std::map<size_t, ExpressionRunnerRef> expressionRunners;
```

Comment:
Yikes, it looks like the Binaryen.js tests are not actually being run @sbc100 @kripken

Reply:
Looks like maybe a bug in  `./scripts/emcc-tests.sh`?   It seems to be build binaryen.js but then not running actual tests.

Reply:
Oops, that was me #2734

Reply:
Fix: https://github.com/WebAssembly/binaryen/pull/2810